### PR TITLE
perf: use lodash subutils

### LINKED ISSUE #44: refactor: replace lodash module with its subutils
image

### PR DESCRIPTION
**Pull request by oljekechoro**
closes #44

### CODE REVIEW

File: packages/common/package.json [hunk 1]
```diff
@@ -36,15 +36,23 @@
...
+    "lodash.invert": "^4.3.0",
+    "lodash.set": "^4.3.2",
+    "lodash.unset": "^4.5.2",
+    "tslib": "^2.3.0",
+    "xregexp": "^5.0.2"
   },
   "devDependencies": {
-    "@qiwi/masker-infra": "1.0.3"
+    "@qiwi/masker-infra": "1.0.3",
+    "@types/lodash.clonedeep": "^4.5.6",
```
Comment (antongolub) [inline packages/common/package.json:51]: types should follow its deps → move to "dependencies"

File: packages/common/package.json [hunk 2]
```diff
@@ -36,15 +36,23 @@
...
-    "@types/lodash": "^4.14.170",
-    "xregexp": "^5.0.2",
     "@types/xregexp": "^4.3.0",
-    "flattie": "^1.1.0"
+    "flattie": "^1.1.0",
+    "lodash.clonedeep": "^4.5.0",
+    "lodash.get": "^4.4.2",
+    "lodash.invert": "^4.3.0",
+    "lodash.set": "^4.3.2",
+    "lodash.unset": "^4.5.2",
```
Comment (antongolub) [inline packages/common/package.json:45]: is this need for common? Seems like it deps of `packages/schema`

**Review by antongolub — CHANGES_REQUESTED** [review]

File: packages/common/package.json (hunk 2 above)
Comment (oljekechoro) [reply packages/common/package.json:45]: lodash was installed in common

Comment (antongolub) [reply packages/common/package.json:45]: But these methods are not required, right?

**Review by antongolub — APPROVED** [review]

**Review by antongolub — APPROVED** [review]

**Pull request merged**